packages/tentales/src/middleware/middleware-utils.ts

```diff
@@ -7,9 +7,7 @@ function getMiddlewaresAtHook(
   return hookMiddlewares[hookName] as Middleware[]
 }
 
-export function getHookMiddlewaresFromConfig(
-  config: Config,
-): Hook[] {
+export function getHookMiddlewaresFromConfig(config: Config): Hook[] {
```

Comment:
I think this can be renamed to simply `getHookMiddlewares`, the config part is kinda implied by the fact that you pass it the `Config` interface

Reply:
Yeah... I'm to deep in js-land